src/accsr/remote_storage.py

```diff
@@ -320,12 +320,16 @@ def size_files_to_sync(self) -> int:
         :return: the total size of all local objects that need synchronization if self.sync_direction='push' and
             the size of all remote files that need synchronization if self.sync_direction='pull'
         """
-        if self.sync_direction not in ["push", "pull"]:
-            raise RuntimeError(
-                "sync_direction has to be set to push or pull before computing sizes"
-            )
-        mode = "local" if self.sync_direction == "push" else "remote"
-        return _get_total_size(self.files_to_sync, mode=mode)
+        assert self.sync_direction is not None
```

Comment:
Pls add the message from the runtime error here

Reply:
My recent update which simplified the interfaces has removed the need for this assertion.

Reply:
Please review this commit: https://github.com/appliedAI-Initiative/accsr/pull/32/commits/1570c0f58f1dbd03aa2a833263dd8abde25f67d2

Reply:
Nice!